### ISSUE TITLE
Submitting translation after send back, creates a review task assigned to translator

### ISSUE THREAD

**Issue by ehazlett** [pculture/unisubs#247]
Although we can't reproduce it on staging, translation tasks that are submitted after being sent back are creating review tasks assigned to the translator, which causes difficulty, and creates crediting errors. This has happened with recently added videos on the TED team. We would assume these have a relatively clean revision history, and wouldn't produce issues with the send back feature. Since these sounded newer, and more unique, I wanted to pass them for further investigation. The send back feature is a high priority issue for TED.

There are two situations here, with exact outcomes:

Situation #1
==Amara URL's==
- Talk info: Matt Mills: Image recognition
- Language: Japanese
- Revision History: http://www.universalsubtitles.org/en/videos/k4QdIGaytrRh/ja/355980/218463/#revisions
- Tasks in admin: http://www.universalsubtitles.org/en/admin/teams/task/?q=matt+mills
- Video page: http://www.universalsubtitles.org/en/videos/k4QdIGaytrRh/info/matt-mills/

==Whats happening==
- 1) Translator (DSK INOUE) creates 4 revisions while working on translation and submits.
- 2) Reviewer (Yasushi Aoki) sends back to translator.
- 3) Translator edits translation (creates revision 5), and submits it.
- 4) Reviewer edits translation a few times, and sends it back to the translator (creating revisions 6, 7, and 8)
- 5) (waiting on confirmation for this step) The translation task is submitted by translator, and a review task is created for the translator, instead of the reviewer.

==What we expect==
- That in step 5, the translator never is assigned a review task. And that after submitting the translation, the review is assigned to the reviewer.

Situation #2
==Amara URL's==
- Talk info: Karen Bass: Unseen footage untamed nature
- Language: Japanese
- Revision History: http://www.universalsubtitles.org/en/videos/fec7c69d71ceb7723f7382e2830cad85866dc851/ja/340518/#revisions
- Video Page: http://www.universalsubtitles.org/en/videos/4tuFjy0DGPbF/info/karen-bass-unseen-footage-untamed-nature/
- Admin tasks: http://www.universalsubtitles.org/en/admin/teams/task/?q=karen+bass&all=

==What's happening==
- 1) Mieko Akai did the translate task, and made revisions 0-6
- 2) Yasushi Aoki did the review made revision 7, 8, and 9 and sent back
- 3) Mieko Akai submitted the resulting translate task without modifying the subtitles and a review task was assigned to Mieko.

==What we expect==
- That in step 3, the review task is assigned to Yasushi, and that Mieko is never assigned a review task.

**Comment by kslottow** [comment]
we won't fix this one because it's specific to the task model and should not occur with subtitle requests in the collab model.
